GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/ContentTypes/accordion.config

```diff
@@ -43,10 +41,10 @@
   </GenericProperties>
   <Tabs>
     <Tab>
-      <Key>d0bdf898-e41c-4b33-9465-959e7d7a72ff</Key>
+      <Key>209eaccb-9deb-4850-b0a4-b2cf243b64e5</Key>
       <Caption>Content</Caption>
       <Alias>content</Alias>
-      <Type>Tab</Type>
+      <Type>Group</Type>
```

Comment:
This changed from Group to Tab. Your outdated content is moving it back, which is not wanted.